locales/en.default.schema.json

```diff
@@ -1909,7 +1909,11 @@
       }
     },
     "main-login": {
-      "name": "Login"
+      "name": "Login",
+      "shop_login_button": {
+        "name": "Sign in with Shop Settings",
+        "hide": "Hide button"
```

Comment:
Yeah, the footer example is just a reference, we are moving away from being extra detailed, but these existing settings haven't been following those guidelines yet. The footer also has a lot of various settings so the headers are more important there. Which wouldn't be for the login section because there is only a single setting except the padding ones 😄 

So we would be still good with no header in this use case IMO! 🙌

Reply:
I'm slightly concerned about dropping the header because we wouldn't want padding settings to get confused with padding for the button. Also, the same requirement for the follow button (_Pay must be enabled for the button to be displayed_) is also true for the login button. 

Because of that, I'd thought that I might need to **add** to the information for the login button rather than take away. However, I'm open to following your recommendation since this isn't my area of expertise :)

Reply:
Awesome, thank you both so much for the feedback! I'll make the changes :)